packages/sdk/react-surface/src/Plugin.ts

```diff
@@ -4,37 +4,92 @@
 
 import { FC, PropsWithChildren } from 'react';
 
+/**
+ * Props passed to a component by the `Surface` resolver.
+ */
 export type ComponentProps = PropsWithChildren<{
```

Comment:
Rename PluginComponentProps?

Reply:
Doesn't feel entirely right, but probably better than what it currently is